src/pages/references/stacks-cli.md

```diff
@@ -27,6 +28,30 @@ You must have [Node.js](https://nodejs.org/en/download/) v12 or higher (v14 reco
 
 To install the command line, do the following:
 
+### Step 1: [Download or `git clone` the command line repository code](https://github.com/blockstack/stacks.js/tree/master/packages/cli)
```

Comment:
I think we should just delete the clone repo instructions and use `npm install -g @stacks/cli` here.

Reply:
good point! I didn't look closely enough. will us npm instead